src/Microsoft.TemplateEngine.Orchestrator.RunnableProjects/SymbolModel/ParameterSymbol.cs

```diff
@@ -141,5 +152,82 @@ internal static ParameterSymbol FromDeprecatedConfigTag(string name, string valu
 
             return symbol;
         }
+
+        private static TemplateParameterPrecedence GetPrecedence(bool isRequired, JObject jObject)
+        {
+            string? isRequiredCondition = ParseIsRequiredConditionField(jObject);
+
+            // Initialize IsEnabled - as a condition or a constant
+            string? isEnabledCondition = null;
+            bool isEnabled = true;
+            if (jObject != null && jObject.TryGetValue("IsEnabled", StringComparison.OrdinalIgnoreCase, out JToken? isEnabledToken))
+            {
+                if (isEnabledToken!.TryParseBool(out bool enabledConst))
+                {
+                    isEnabled = enabledConst;
+                }
+                else if (isEnabledToken.Type == JTokenType.String)
+                {
+                    isEnabledCondition = isEnabledToken.ToString();
+                }
+            }
```

Comment:
should `ParseIsRequiredConditionField` be used instead? (rather `ParseStringOrBoolField`) - may be moved to `JExtensions` instead